src/tests/ftest/util/osa_utils.py

```diff
@@ -236,65 +299,54 @@ def run_ior_thread(self, action, oclass, test):
                                    kwargs={"pool": self.pool,
                                            "oclass": oclass,
                                            "test": test,
-                                           "flags": flags})
+                                           "flags": flags,
+                                           "single_cont_read":
+                                           single_cont_read,
+                                           "fail_on_warning":
+                                           fail_on_warning})
         # Launch the IOR thread
         process.start()
         # Wait for the thread to finish
         process.join()
 
-    def ior_thread(self, pool, oclass, test, flags):
+    def ior_thread(self, pool, oclass, test, flags,
+                   single_cont_read=True,
+                   fail_on_warning=True):
```

Comment:
The method description seems off. It seems this method starts just one IOR process and it doesn't wait anything.

Reply:
Done.